docs/components/textarea.md

```diff
@@ -31,22 +31,38 @@ If the label is omitted it can be handled with another component, like **KLabel*
 
 ## Props
 ### Size
+
 You can specify `rows`, `cols` for the textarea size.
 
-<KTextArea :rows=2 :cols=15 placeholder="rows:2, cols:15" />
-<br>
 <KTextArea label="Size" :rows=3 :cols=20 placeholder="I'm labelled and customized!" />
 <br>
 <KTextArea :rows=8 :cols=25 placeholder="rows:8, cols:25"  />
 
 ```vue
-<KTextArea :rows=2 :cols=15 placeholder="rows:2, cols:15" />
-<br>
 <KTextArea label="Size" :rows=3 :cols=20 placeholder="I'm labelled and customized!" />
 <br>
 <KTextArea :rows=8 :cols=25 placeholder="rows:8, cols:25"  />
 ```
 
+### characterLimit
+
+Use this prop to specify a character limit for the textarea, defaults to `2048`.
+
+<KTextArea :characterLimit="500" />
+
+```vue
+<KTextArea :characterLimit="500" />
+```
+
+### disableCharacterLimit
+
+Use this prop to remove the character limit on the textarea. Defaults to `false`.
+
+<KTextArea :disableCharacterLimit="true" />
+
+```vue
+<KTextArea :disableCharacterLimit="true" />
```

Comment:
Unless the standard is to explicitly show props with a `true` value, this could just be written as

``` html
<KTextArea disableCharacterLimit />

<!-- or -->

<KTextArea disable-character-limit />
```

Reply:
We don't really have a standard, and do a mix throughout the docs, but I went ahead and changed this instance.